src/JWT.Extensions.AspNetCore/JWT.Extensions.AspNetCore.csproj

```diff
@@ -15,6 +15,8 @@
     <AssemblyVersion>6.0.0.0</AssemblyVersion>
```

Comment:
Please bump Version for all 3 packages but don't bump the other 2 attributes.

Reply:
Should I make it 6.1.0.0? Not sure if you'd consider this a breaking change. It could break someone who is manually referencing/loading the assembly or otherwise relying on the assembly's full name.

Reply:
Let's keep 6.0.0.0, I think that will be fine.